docs/src/input.md

```diff
@@ -415,8 +409,7 @@ page.locator('button#submit').dispatch_event('click')
 ```csharp
 await page.Locator("button#submit").DispatchEventAsync("click");
 ```
-
-### API reference
+#### API reference
```

Comment:
Replace "via simply dispatching a click event on the element" with "via simply dispatching a click event on the element with [`method: Locator.dispatchEvent`]"